parser/base_scraper.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABC, abstractmethod
+from parser.apartment import Apartment
+
+
+class BaseScraper(ABC):
+
+    def __init__(self):
+        self.apartments_data = []
```

Comment:
I've already suggested you making parser stateless. Stateless means that it does not have any internal state. By state i mean something that will characterize the instance of this class in some time period. `apartment_data` attribute is an example of such behavior.